### DISCUSSION TITLE
"Class 'ConvectionTerm' does not have any field names '_fun'" Error

### DISCUSSION THREAD

**Discussion by AnthonyB08** [dpploy/engy-5310#85] [Q&A] [answered]
Hi, 

Can anyone explain what this error is? I mimicked other variable implementation, such as normalBC, so there should be no issue but I am getting this compile error. 

![image](https://user-images.githubusercontent.com/55299169/118141503-6cd26380-b3d7-11eb-9be7-7815d1e1b289.png)

![image](https://user-images.githubusercontent.com/55299169/118141590-84a9e780-b3d7-11eb-847c-3c80de3b1d9c.png)

P.S yes I took this this from Austin, because I want to see any changes if MOOSE doesn't solve the Momentum Equation

**Comment by AnthonyB08** [comment]
Could this be the source of my error ? 

![image](https://user-images.githubusercontent.com/55299169/118150450-90e67280-b3e0-11eb-90fd-a04603a352a4.png)

**Comment by AnthonyB08** [comment, marked as answer]
Yes this was the source of error. 

For those continuing moose, do not build another project within the same folder. 

I.E I had a test folder built inside my main tritium folder

**Reply by dealmeidavf** [reply]
You should be able to just enter a project area and  build there once you have the whole course repo.

**Reply by AnthonyB08** [reply]
When compiling it read two header files that were overwriting one another because I had been running simulations in both. Moving the header files and source, and recompile fixed the declaration of variables.
